src/structures/VoiceChannel.js

```diff
@@ -46,7 +46,12 @@ class VoiceChannel extends BaseGuildVoiceChannel {
    * @readonly
    */
   get speakable() {
-    return this.permissionsFor(this.client.user).has(Permissions.FLAGS.SPEAK, false);
+    const permissions = this.permissionsFor(this.client.user);
+    if (!permissions) return false;
+    // This flag allows speaking even if timed out
+    if (permissions.has(Permissions.FLAGS.ADMINISTRATOR, false)) return true;
+    if (!permissions.has(Permissions.FLAGS.SPEAK, false)) return false;
+    return this.guild.me.communicationDisabledUntilTimestamp < Date.now();
```

Comment:
How are timeouts related to speaking in a voice channel if you can’t join it to begin with?

Reply:
Administrator can receive timeout  however they are still able to join VC

Reply:
In that case they can also speak in the vc